master/internal/trial.go

```diff
@@ -344,6 +346,9 @@ func (t *trial) runningReceive(ctx *actor.Context) error {
 
 	case killTrial:
 		if t.task != nil {
+			if msg.timedOut {
+				ctx.Log().Info("killing hanging trial after timeout")
```

Comment:
nit: "killing unresponsive trial after timeout", it sounds a bit odd to use an -ing adjective after an -ing verb